js/main.js

```diff
@@ -118,6 +131,37 @@ function setStartup(lStartup){
     });
 }
 
+// Method to overwrite user config on mac installer
+function updateUserConfigMac() {
```

Comment:
I tried implementing this with the `node's fs` module and the following is the issue that I faced.
 
The installer runs as a root process where the directory created will be with a different permissions so to change the permissions using the method `fs.chown(path, uid, gid, callback)` requires `uid - integer` & `gid - integer` but we don't get these values if the process is running in root.

**`Root User`**
![screen shot 2017-07-14 at 8 57 33 pm](https://user-images.githubusercontent.com/13243259/28219965-7925a138-68db-11e7-820a-ac118a7edc69.png)

**`User`**
![screen shot 2017-07-14 at 8 58 23 pm](https://user-images.githubusercontent.com/13243259/28220021-9e16986c-68db-11e7-8ce0-4285c4055f5c.png)